schema/changelog-4.4b.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd"
+  logicalFilePath="changelog-4.4b">
+
+  <changeSet author="author" id="changelog-4.4b-all">
+    <sql>
+      ALTER TABLE tc_users
+      ADD totpKey VARCHAR(128)
+    </sql>
+    <sql>
+      ALTER TABLE tc_users
+      ADD useTotp BOOLEAN DEFAULT FALSE;
```

Comment:
Already commented on web PR. I think single field is enough.

Reply:
This field is there when in the server configuration the totp preference is user instead of all. The 'user' settings means we can turn totp on per user while 'all' means it is on for every user it cannot be turned off.
If we only want to have the 'user' option than this checkbox can go.

Reply:
Let's continue discussion in a single place. Please don't duplicate comments.